app/templates/search/_categories.html

```diff
@@ -1,14 +1,47 @@
-<ul>
-  {% for category in categories %}
-    <li>
-      {% if category.link %}<a href="{{ category.link }}">{% endif %}{% if category.selected %}<strong>{% endif %}
-        {{ category.label }}
-      {% if category.selected %}</strong>{% endif %}{% if category.link %}</a>{% endif %}
-      {% if category.children and category.selected %}
-        {% with categories=category.children %}
-          {% include 'search/_categories.html' %}
-        {% endwith %}
+{%- macro top_level_link(text, url) -%}
+  <a href="{{ url }}" class="lot-filters__top-level-link"><span class="visually-hidden">Back to </span>{{ text }}</a>
```

Comment:
Removing the "Back to" text for now as Voiceover is pretty good about describing how the list is structured. When @carolinegreen is back, we can see what she thinks.